test/source/tests/decrypt.ts

```diff
@@ -1151,13 +1151,16 @@ XZ8r4OC6sguP/yozWlkG+7dDxsgKQVBENeG6Lw==
           await page.waitAll('iframe');
           const pgpBlock = await page.getFrame(['pgp_block.htm']);
           await pgpBlock.waitForSelTestState('ready');
+          const expectedPrintDateString = new Date(1712818847000).toLocaleString();
           const printPage = await browser.newPageTriggeredBy(t, () => pgpBlock.click('@action-print'));
+          await printPage.waitForContent('@print-date', expectedPrintDateString);
           await printPage.waitForContent('@print-user-email', 'First Last <flowcrypt.compatibility@gmail.com>');
           await printPage.waitForContent('@print-subject', '<b><h1>Test print dialog');
           await printPage.waitForContent('@print-from', 'From: sender@domain.com');
           await printPage.waitForContent('@print-to', 'To: flowcrypt.compatibility@gmail.com');
           await printPage.waitForContent('@print-cc', 'ci.tests.gmail@flowcrypt.dev');
           await printPage.waitForContent('@print-content', 'Test print message');
+          expect(!isNaN(Date.parse(await printPage.readHtml('@print-date')))).to.equal(true);
```

Comment:
this line isn't needed anymore, as we're checking for correct date in `print-date` earlier in this test

Reply:
Agree. I have removed it away from the other test.